Source/Bugsnag.h

```diff
@@ -36,6 +36,7 @@
 #import "BugsnagDeviceWithState.h"
 #import "BugsnagEndpointConfiguration.h"
 #import "BugsnagError.h"
+#import "BugsnagErrorTypes.h"
```

Comment:
This import is required as `Bugsnag.h` is an umbrella header. This means it needs to import all public header files so that users can access them when importing the module.